Automattic/jetpack
# Jetpack Boost: Add schema validation

This ensures that Critical CSS state is always the same shape both before saving to the db and after reading it. It also makes it easier to see at a glance what the shape of the data is.

## Proposed changes:
Implements a new class called Schema and a couple of pre-defined rules that we can use to leverage schemas.


### Other information:

## Jetpack product discussion
N/A

## Does this pull request change what data or activity we track or use?
N/A

## Testing instructions:
1. Apply patch
2. Create a schema
3. Validate the schema
4. Ensure that results match expectations
